lib/tests/streamlit/dataframe_util_test.py

```diff
@@ -370,6 +369,13 @@ def test_data_frame_with_unsupported_column_types(self):
                 f"Unsupported types of this dataframe should have been automatically fixed: {ex}"
             )
 
+    def test_is_pandas_data_object(self):
+        """Test that `is_pandas_data_object` correctly detects pandas data objects."""
+        assert dataframe_util.is_pandas_data_object(pd.DataFrame()) is True
+        assert dataframe_util.is_pandas_data_object(pd.Series()) is True
+        assert dataframe_util.is_pandas_data_object(pd.Index(["a", "b"])) is True
+        assert dataframe_util.is_pandas_data_object(pd.array(["a", "b"])) is True
```

Comment:
nit: I think it would be beneficial if you also had a test to check for `is False`. Otherwise, the function could always return `True` for any input 😅